### DISCUSSION TITLE
Release hotfix 🚀

### DISCUSSION THREAD

**Discussion by hgoabreu** [kafbat/kafka-ui#501] [General]
Hey 👋

After merging https://github.com/kafbat/kafka-ui/pull/378 container images are now fully running in arm systems, we should release (at least) a hotfix in order to push these changes to ghcr.

Is it possible to trigger one (or a minor)?

**Comment by p5** [comment]
Would also like a hotfix, but for a separate issue.
v1.0.0 fails to startup when using the Serde library, but this is resolved in `main`.
This is the only blocker for us adopting the new and improved Kafka UI fork!

**Comment by Haarolean** [comment]
all the changes you might need are present in main and every push to one has a corresponding build, you can see all the tags here:
https://github.com/kafbat/kafka-ui/pkgs/container/kafka-ui
Also you could pull `main`-tagged image instead.

**Reply by p5** [reply]
While that is an option to hobbyists, this will likely be blocked an organization's security team immediately.  For us, we must be pinned to a specific version that they have vetted and given the "okay" to.  It's far easier getting this approval for a :v1.0.1@shaxyz rather than for a :main@shaxyz

**Reply by hgoabreu** [reply]
This is mainly the reason. As `main` tag can change in every deployment and we might not be aware of breaking changes / CVE scans.

**Reply by Haarolean** [reply]
That's why I suggested two options, one doesn't rely on main tag

**Reply by hgoabreu** [reply]
Sounds good @Haarolean 👍 
Is there a roadmap / dicussions (or open issues) to start relying on [semver](https://semver.org/) for the CI?

**Reply by Haarolean** [reply]
@hugomiguelabreu 

What makes you think we do not? We're following semver, the latest release is 1.0, the next one is 1.1, see our [roadmap](https://github.com/orgs/kafbat/projects/11)

**Reply by hgoabreu** [reply]
Oh I am sorry @Haarolean 🙇, indeed I wasn't aware of the roadmap for the `1.1`.  I though, since there were no hotfixes, there wasn't a semantic versioning process.

Looks good! Thanks!
